extension/options.js

```diff
@@ -33,35 +31,28 @@
 		}
 
 		formRootUrl.addEventListener('change', function () {
-			var url = normalizeRoot(formRootUrl.value) + 'settings/tokens/new?scopes=notifications';
-			ghSettingsUrl.href = url;
-		});
-
-		formUseParticipating.addEventListener('change', function () {
-			GitHubNotify.settings.set('useParticipatingCount', formUseParticipating.checked);
+			var url = normalizeRoot(formRootUrl.value);
+			var urlSettings = normalizeRoot(formRootUrl.value) + 'settings/tokens/new?scopes=notifications';
+			// case of url is empty: set to default
+			if (url === normalizeRoot('')) {
+				GitHubNotify.settings.remove('rootUrl');
+				url = GitHubNotify.settings.get('rootUrl');
```

Comment:
Because the only way to fetch the default value is the "GitHubNotify.settings.get" method. No?